vid/src/advz/test.rs

```diff
@@ -48,6 +48,28 @@ fn commit_only_timer() {
     let _ = advz.commit_only(payload_random);
 }
 
+#[ignore]
+#[test]
+#[cfg(feature = "gpu-vid")]
+/// Stress test with varied payload sizes for GPU memory
+/// leakage/fragmentation.
```

Comment:
how does this test leakage or fragmentation?

iiuc, this only put lots of running instances, and in case we didn't clean up (after use) the memory, it would overflow the memory?

but how does it test fragmentation?

Reply:
Suggested by Salman. It really depends on the GPU memory management. Hypothesis is that if you make many GPU memory allocations and they are not cleaned up fast enough, then there're potentially fragmentations that prevent you from allocating even if the GPU memory is not full.

Reply:
plz put this explanation in a comment 🙏